cli/commands/commands.go

```diff
@@ -46,6 +47,7 @@ func AddCommands(cmd *cobra.Command) {
 		metric.NewMetricCommand(),
 		tool.NewToolCommand(),
 		assessment.NewAssessmentCommand(),
+		orchestrator.NewOrchestratorCommand(),
```

Comment:
Hmm the only issue I have with this, is that all the other commands are structured according to the resource and not the service one retrieves it from. Metric and tool information are also accessed by the orchestrator. Would it make sense to just replace the assessment list commands with those in the orchestrator? Do we need to retrieve the list of assessment results from the assessment?

Reply:
It definitely makes sense if the orchestrator is deployed as well. I always think in terms when somebody want to use one service, e.g. assessment, exclusively. But is that even realistic?

Reply:
Ok, since we won't use the CLI in deployment, I guess it doesn't matter anyway? To verify that we are on the same page: you want to keep the `cl assessment list` command but it should point to the Orchestrator method?

Reply:
> Ok, since we won't use the CLI in deployment, I guess it doesn't matter anyway? To verify that we are on the same page: you want to keep the `cl assessment list` command but it should point to the Orchestrator method?

That would be "in line" with the previous commands, yes. If this really makes sense: not sure